packages/tools/kadena-cli/src/utils/createCommand.ts

```diff
@@ -180,7 +184,9 @@ export function createCommand<
           console.log('\n');
         }
 
-        await action(config, newArgs);
+        const generalArgs = rest.flatMap((r) => r.args);
+
+        await action(config, generalArgs, newArgs);
```

Comment:
I've discussed with @nillo. The third argument "newArgs" can just go away, as we don't use it currently.